server/tests/cluster.rs

```diff
@@ -0,0 +1,104 @@
+use std::time::Duration;
+
+use enumset::enum_set;
+use futures_util::StreamExt;
+use restate_local_cluster_runner::{
```

Comment:
nit: Our convention is to leave a blank line between third-party imports and `restate_` ones. 
```suggestion

use restate_local_cluster_runner::{
```